packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -77,32 +77,28 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
 
     return validData ? (
         <BigNumberContainer ref={containerRef} {...wrapperProps}>
-            {rect?.width ? (
```

Comment:
if `useResizeObserver` does not fire on mount then it will not render have the rect or the width - thus it will not render from here